### DISCUSSION TITLE
Hasil knit dan RMD berbeda

### DISCUSSION THREAD

**Discussion by Talioll** [teamalgoritma/community#473] [Q&A] [answered]
Terdapat 3 chunk dengan hasil yg berbeda dari knit html. Sya sudah mencoba untuk cari solusinya, tapi belum paham.  Apakah ada hubungannya dengan encoding? Mohon dibantu untuk solusinya, terima kasih :)

berikut salah satu chunk yg memiliki perbedaan pada knit:
![error1](https://user-images.githubusercontent.com/112487401/205936758-750fc6e9-47f6-4f51-ae60-7588547c3a9b.jpg)

**Comment by nisa-basalamah** [comment, marked as answer]
Dear @Talioll ,

Ada kemungkinan perbedaan output tersebut muncul karena terdapat bagain kode program yang melakukan random sampling di dalamnya namun belum disertai dengan fungsi `set.seed() ` untuk mengunci hasil sampling tersebut tetap sama setiap kali program dijalankan. Apabila berkenan, Anda dapat melampirkan link menuju versi lengkap dari projek yang sedang dikerjakan untuk dapat dibantu pengecekan lebih lanjut.

Terima kasih

**Reply by Talioll** [reply]
Dear mba nisa, 

untuk data yg dipakai (trx.clean) adalah data frame biasa (data frame asli yg sudah di bersihkan saja), proses data tidak menggunakan sample. Dalam chunk terkait apabila di run berkali2 juga tidak menunjukan perubahan pada hasil, logikanya jika pada data frame tersebut diproses dengan fungsi yg menggunakan random sampling,  chunk tersebut juga akan menampilkan hasil yg terus berbeda jika tidak menggunakan set.seed().  Demikian penjelasan sy, mungkin ada solusi lain?

**Reply by Talioll** [reply]
Namun begitu masukan dari mba nisa akan sy coba terlebih dahulu, sy akan kembali mengupdate apabila masukan dari mba nisa bisa menjadi solusi dari kasus sy, terimakasih telah membantu :)

**Comment by Talioll** [comment]
Dear mba @nisa-basalamah,

terima kasih atas jawabannya, sy lupa kalau k-means menggunakan random sampling. Dengan begitu cluster yg digunakan berbeda ketika sy menambah cluster ke data frame yg akan digunakan. Solusi yg dberikan mbak nisa sudah sy terapkan kepada object k-means. Terima kasih :)
